oq_output/hazard_map_converter.py

```diff
@@ -97,7 +98,10 @@ def parse_nrml_hazard_map(nrml_hazard_map):
         else:
             metadata[option] = None
     if "SA" in metadata["imt"]:
-        metadata["sa_period"] = node_set.attrib['saPeriod']
+        imt_str =  node_set.attrib['IMT']
+        m = re.search('.*\((\d*\.\d*)\).*', imt_str)
```

Comment:
As I am allergic to regular expressions I would just note that the same thing can be done with `period = node_set["IMT"].translate(None, "SA()")` without the need for the regex import